docs/configuring-well-known.md

```diff
@@ -36,6 +36,41 @@ However, this playbook installs your Matrix server on another domain (e.g. `matr
 To learn how to set it up, read the Installing section below.
 
 
+## (Optional) Introduction to Homeserver Admin Contact and Support page
+
+[MSC 1929](https://github.com/matrix-org/matrix-spec-proposals/pull/1929) specifies a way to add contact details of admins, as well as a link to a support page for users who are having issues with the service.
+
+This MSC did not get accepted yet, but we think it might already be useful to Homeserver admins who wish to provide this information to end-users.
+
+The two playbook variables that you could look for, if you're interested in being an early adopter, are: `matrix_homeserver_admin_contacts` and `matrix_homeserver_support_url`.
+
+Example snippet for `vars.yml`:
+```
+# Homeserver admin contacts as per MSC 1929 https://github.com/matrix-org/matrix-spec-proposals/pull/1929
+matrix_homeserver_admin_contacts: |
+  [
+    {
+      "matrix_id": "@admin1:domain.tld",
+      "email_address": "admin@domain.tld",
+      "role": "admin"
+    },
+    {
+      "matrix_id": "@admin2:domain.tld",
+      "email_address": "admin@domain.tld",
+      "role": "admin"
+    },
+    {
+      "email_address": "security@domain.tld",
+      "role": "security"
+    }
+  ]
```

Comment:
This appears to be a string with JSON in it.

At the same time, the `matrix_homeserver_admin_contacts` variable in `roles/matrix-base/defaults/main.yml` is defined as a list (`matrix_homeserver_admin_contacts: []`).

Why not make it an actual list here too?

```suggestion
matrix_homeserver_admin_contacts:
   - matrix_id: @admin1:domain.tld
     email_address: admin@domain.tld
     role: admin
   - matrix_id: @admin2:domain.tld
     email_address: admin@domain.tld
     role: admin
   - email_address: security@domain.tld
     role: security
```

Reply:
Resolved in [32a604d](https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/1931/commits/32a604d7c16b28dad552c8c87be710bdc9e37d92)